pymc/tests/test_sampling.py

```diff
@@ -1212,3 +1212,58 @@ def test_sample_deterministic():
         idata = pm.sample(chains=1, draws=50, compute_convergence_checks=False)
 
     np.testing.assert_allclose(idata.posterior["y"], idata.posterior["x"] + 100)
+
+
+class TestDraw(SeededTest):
+    def test_draw_one_variable(self):
+
+        with pm.Model() as model:
+            x = pm.Normal("x")
+
+        x_draws = pm.draw(x)
+        assert x_draws["x"].shape == (500,)
+
+    def test_draw_one_variable_with_number_draws(self):
+
+        with pm.Model() as model:
+            x = pm.Normal("x")
+
+        num_draws = 100
+        x_draws = pm.draw(x, draws=num_draws)
+        assert x_draws["x"].shape == (num_draws,)
+
+    def test_draw_variables(self):
+
+        with pm.Model() as model:
+            x = pm.Normal("x")
+            y = pm.Normal("y", shape=10)
+            z = pm.Uniform("z", shape=5)
+
+        num_draws = 1000
+        # Draw samples of a list variables
+        draws = pm.draw([x, y, z], draws=num_draws)
+        assert draws["x"].shape == (num_draws,)
+        assert draws["y"].shape == (num_draws, 10)
+        assert draws["z"].shape == (num_draws, 5)
+
+        # Draw samples of a tuple variables
+        draws = pm.draw((x, y, z), draws=num_draws)
+        assert draws["x"].shape == (num_draws,)
+        assert draws["y"].shape == (num_draws, 10)
+        assert draws["z"].shape == (num_draws, 5)
+
+    def test_multivariate(self):
+        with pm.Model() as model:
+            mln = pm.Multinomial("mln", n=5, p=np.array([0.25, 0.25, 0.25, 0.25]))
+
+        mln_draws = pm.draw(mln, draws=100)
+        assert mln_draws != (100, 4)
+
+    def test_shape_edgecase(self):
```

Comment:
Do we need this test?

Reply:
Hi, may be not now, it will be removed :)